docroot/modules/custom/mass_content/src/Commands/MassContentCommands.php

```diff
@@ -366,6 +366,84 @@ public function healRawLinkInContent($options = ['format' => 'table']): RowsOfFi
     return new RowsOfFields($rows);
   }
 
+  /**
+   * Populate contextual search related field default values.
+   *
+   * @command mass-content:populate-search-fields
+   *
+   * @aliases mpsf
+   */
+  public function populateContextualSearchFields() {
+    // 1. Log the start of the script.
+    $this->logger()->info('Update nodes batch operations start');
+
+    // 2. Retrieve the nodes.
+    $storage = $this->entityTypeManager->getStorage('node');
+    try {
```

Comment:
Why do we have a try here? There is just array operations below. Maybe this is refactoring cruft.

Reply:
Yes, removed.